File: framework/scripts/mutants/2/FieldAccess.java
public class FieldAccess {
    public int x = 0;
    public int y = 0;
    FieldAccess fa = this;

    int test(int ex, int y) {
        int a = x;
        int b = y;
        int c = fa.fa.y;

        x = ex;
        ;
        this.y = b;

        int result = a + b + c;
        return result;
    }

}

File: serializer/pegnodes/src/pegnodes/tests/field_access.clj
(ns pegnodes.tests.field-access
  (:require  [clojure.test :as t])
  (:require  [pegnodes.pegs :refer :all])
  (:require  [pegnodes.tests.tests :refer :all]))

(def file-path (str subjects-directory "/field-access/FieldAccess.java"))
(test-file file-path)
